recipes/tbl2asn/meta.yaml

```diff
@@ -1,15 +1,18 @@
+{% set version = "25.8" %}
+{% set release_date = "2023-01-19" %}
+
 package:
   name: tbl2asn
-  version: "25.7"
+  version: {{ version }}
 
 source:
-  - url: ftp://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/tbl2asn/mac.tbl2asn.gz  # [osx]
-    sha256: 79b0f0a764235656bbaec0541bd03d87514b658dfe319ea94b8090d1dfd3174f  # [osx]
-  - url: ftp://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/tbl2asn/linux.tbl2asn.gz  # [linux]
-    sha256: f83a817e62854260739c170996b8ee61fb8e0317aded4b4818849ff91eba7c05  # [linux]
+  - url: https://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/versions/{{ release_date }}/by_platform/mac/tbl2asn.mac.gz  # [osx]
```

Comment:
I assume those are ARM-based binaries and those will not work here. Thats why we prefer to compile from source.